src/components/Email/EmailTemplates/AdminApprovalProcessEmail.jsx

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Email } from 'react-html-email';
+
+import {
+  BoxApprovedEmailPicture,
+  BoxRejectedEmailPicture,
+  ChangesRequestedEmailPicture,
+  FYATextLogo,
+} from '../../../common/utils';
+
+// The type of email sent depends directly on the 'type' prop passed in ('approved', 'changes requested', or 'rejected')
+// Inline styling is necessary because it won't detect styles from a separate css page :D
+const AdminApprovalProcessEmail = ({ type, changesRequested, rejectionReason }) => {
+  // Header text
+  const headerDict = {
+    approved: 'Your box launch has been approved!',
+    'changes requested': 'Your box launch requires changes.',
+    rejected: 'Your box launch could not be added.',
+  };
+  // Text below the header
+  const headerSubtextDict = {
+    approved: '',
+    'changes requested':
+      "We're so close! We just need a little more information from you in order to add box to the FYA Launch Map.",
+    rejected: "We're sorry, we couldn't add your box to the FYA Launch Map",
+  };
+  // Text below "What does this mean"
+  const whatDoesThisMeanDict = {
+    approved: (
+      <p style={{ margin: 0, color: 'black' }}>
+        Your box&rsquo;s journey has been added to the FYA Launch Map!
+      </p>
+    ),
+    'changes requested': (
+      <p style={{ margin: 0, color: 'black' }}>
+        We&rsquo;re missing some information to verify your box. Please follow the steps below to
+        complete the process, so we can get it added to the FYA Launch Map!
+      </p>
+    ),
+    rejected: (
+      <>
+        <p style={{ margin: 0, color: 'black' }}>
+          For some reason we were not able to verify your box and as a result, could not add it to
+          the FYA Launch Map.
+        </p>
+        <br />
+        <p style={{ margin: 0, color: 'black' }}>
+          If you believe this is an error and the box should be added, please email us at{' '}
+          <span
+            style={{
+              'font-weight': 700,
+            }}
+          >
+            hello@findyouranchor.us
+          </span>
+        </p>
+      </>
+    ),
+  };
+  // Next steps list
+  const nextStepsDict = {
+    approved: [
+      <li key={0}>
+        Find your box on the Launch Map{' '}
+        <a
+          href={`${process.env.REACT_APP_BASE_URL}`}
+          style={{
+            'font-weight': 700,
+          }}
+        >
+          here
+        </a>
+      </li>,
+      <li key={1}>Share your launch on social media!</li>,
+      <li key={2}>Soak in all the good karma for helping us spread more love!</li>,
+    ],
+    'changes requested': [
+      <li key={1}>
+        Resubmit your box request including the missing information{' '}
```

Comment:
We shouldn't be asking the user to resubmit the box form here. I remember discussing this in a project leads/shadowees meeting once, but I don't remember what the decision was about this - could we confirm with design again?

Reply:
As discussed earlier, this and the rejection email will be edited in another PR once the new designs are finalized.